package.json

```diff
@@ -29,5 +29,9 @@
     "neostandard": "^0.12.1",
     "prettier": "^3.5.3",
     "typescript": "^5.8.2"
+  },
+  "dependencies": {
+    "ethers": "^6.13.5",
```

Comment:
I believe `ethers` is not used in this pull request, can we remove this dependency?